WordPress/src/main/java/org/wordpress/android/ui/CommentFullScreenDialogFragment.kt

```diff
@@ -95,12 +118,14 @@ class CommentFullScreenDialogFragment : Fragment(), CollapseFullScreenDialogCont
         private const val EXTRA_REPLY = "EXTRA_REPLY"
         private const val EXTRA_SELECTION_START = "EXTRA_SELECTION_START"
         private const val EXTRA_SELECTION_END = "EXTRA_SELECTION_END"
+        private const val EXTRA_SITE_MODEL = "EXTRA_SITE_MODEL"
 
-        fun newBundle(reply: String, selectionStart: Int, selectionEnd: Int): Bundle {
+        fun newBundle(reply: String, selectionStart: Int, selectionEnd: Int, mSite: SiteModel): Bundle {
```

Comment:
We now prefer to pass the `id` instead of the full object. Passing an object in a `Bundle` can easily lead to crashes [like this](https://github.com/wordpress-mobile/WordPress-Android/pull/10240). `Bundle` has a limit:

> When sending data via an intent, you should be careful to limit the data size to a few KB. Sending too much data can cause the system to throw a TransactionTooLargeException exception.

_from: https://developer.android.com/guide/components/activities/parcelables-and-bundles_

It's generally good practice to use scalar values whenever possible. 

In this case, I think we can pass `SITE_ID` and load the `SiteModel` instead. 🙂